src/services/session-title.mjs

```
@@ -0,0 +1,267 @@
+import {
+  getOpenRouterAttributionHeaders,
+  resolveProviderRequestShapingId,
```

ESLint

[error] 'resolveProviderRequestShapingId' is defined but never used.

(no-unused-vars)